### PR TITLE
add key property to lint message elements

### PR DESCRIPTION
**Pull request by omnikrll**
fixes https://github.com/processing/p5.js-web-editor/issues/62

### CODE REVIEW

File: client/modules/IDE/components/EditorAccessibility.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ class EditorAccessibility extends React.Component {
     if (this.props.lintMessages.length > 0) {
       for (let i = 0; i < this.props.lintMessages.length; i++) {
```
Comment (catarak) [inline client/modules/IDE/components/EditorAccessibility.js]: I didn't notice this earlier, but these days in JS Land I think it's better practice to use forEach than a for loop.

**Comment by catarak** [comment]
i've been meaning to fix this, so ty!

File: client/modules/IDE/components/EditorAccessibility.js [hunk 2]
```diff
@@ -7,18 +7,19 @@ class EditorAccessibility extends React.Component {
...
+            {lintMessage.message}
           </li>
         );
-      }
+      });
     } else {
       messages.push(
-        <p tabIndex="0"> There are no lint messages </p>
+        // no array index so let's just call it 0
+        <p tabIndex="0" key={0}> There are no lint messages </p>
```
Comment (omnikrll) [inline client/modules/IDE/components/EditorAccessibility.js]: Is there a reason this is a <p> and not a <li>, if it's going to go inside a <ul>?

Comment (catarak) [inline client/modules/IDE/components/EditorAccessibility.js]: @MathuraMG do you know why? Does this affect the output of the screen reader?

Comment (MathuraMG) [inline client/modules/IDE/components/EditorAccessibility.js]: I initially did it in hope that the screen reader would read it differently, but it didn't. I believe I forgot to fix it back.
We can change it to `<li>`

**Pull request merged**